src/Symfony/Bundle/SecurityBundle/DataCollector/SecurityDataCollector.php

```diff
@@ -219,6 +221,7 @@ public function reset()
 
     public function lateCollect()
     {
+        $this->data['authenticator_manager_enabled'] = $this->authenticatorManagerEnabled;
```

Comment:
FYI doesn't works if this line is in `collect`

Reply:
what do you mean by "doesn't work" ?

Reply:
it- seems that `collect()` is not called when we render the profiler panel.

Reply:
That is correct, data collectors are meant to collect the state in the request. The profiler panel then loads the state for a request you want to display information.

So this should be moved to `collect()` afaics.

Reply:
>So this should be moved to collect() afaics.

Done 

>what do you mean by "doesn't work"?

Maybe it came from the way that I inject the information (parameter). With the current way addArgument it work like a charm :tada: